product_docs/docs/edbcloud/beta/using_cluster/connecting_your_cluster/01_connecting_from_azure/03_vnet_vnet.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: VNet-VNet Example
+---
+
+These are the steps to connect using VNet-VNet connections. To walk through this example in your own environment, see the prerequisites described in [Connection Walkthrough Examples](../../connecting_your_cluster/#connection-walkthrough-examples).
+
+To use this method, you need to create [Azure VPN Gateways](https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-about-vpngateways) in each connected virtual network. See [Connect Using Azure VPN Gateway Point to Site](#connect-using-azure-vpn-gateway-point-to-site) for details about creating a virtual network gateway. 
+
+!!! Note
+    VPN gateway creation can take up to 45 minutes. 
+
+#### Step 1: Create a VPN Gateway for the EDB Cloud cluster virtual network
+
+ In the **Virtual network gateway** page, create the first VPN gateway for the EDB Cloud Cluster resource virtual network `vnet-japaneast`. Name the VPN gateway "vpng-edbcloud".
+
+ ![](../images/image8.png)
+
+!!! Note
+    The VPN gateway automatically creates a dedicated subnet to accommodate its gateway VMs. Ensure that your EDB Cloud virtual network address space has sufficient IP range for the subnet to prevent errors in the virtual network.
+
+#### Step 2: Create a VPN Gateway for the client VM virtual network
```

Comment:
We talk a lot about "client VMs" in these documents. Should we generalize to "client application", since customers might be using container services or other runtimes?

Reply:
Sure thing, and good point. 
- For the general introduction, we should use "client applications" to make it generically applicable.
- In the content introducing the example, we may call it "client VM" to matching the context.